translations/messages.en.yaml

```diff
@@ -778,3 +778,5 @@ user_badge_moderator: Mod
 user_badge_bot: Bot
 announcement: Announcement
 keywords: Keywords
+deleted_by_moderator: Thread, post or comment is deleted by the moderator.
```

Comment:
I do think we should translate these but I think it should remain more technical than a sentence. reddit/lemmy use `[deleted by moderator]` `[deleted by author]`, I think? Writing a sentence makes it seem no different than a message someone sent

Reply:
I can add it between brackets? It's up to us how we want to handle this message. Currently vintprox was actually confused about this now. Which I understand, hence the reason of this PR.

Reply:
agreed?

Reply:
I'd agree from an accessibility point of view it could be viewed as better compared to reddit / lemmy